client/bindings/python/examples/02_generate_addresses.py

```diff
@@ -5,17 +5,52 @@
 
 # In this example we will create addresses from a mnemonic
 
-secret_manager = MnemonicSecretManager("flame fever pig forward exact dash body idea link scrub tennis minute " +
-    "surge unaware prosper over waste kitten ceiling human knife arch situate civil")
+secret_manager = MnemonicSecretManager("endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river")
 
+# Generate public address with default account index and range.
+addresses = client.generate_addresses(secret_manager, options={})
+
+print('List of generated public addresses:', *addresses, sep='\n')
+print()
 
 # Generate public address with custom account index and range.
-address = client.generate_addresses(secret_manager, {
+addresses = client.generate_addresses(secret_manager, {
+    "accountIndex": 0,
+    "range": {
+        "start": 0,
+        "end": 4,
+    },
+})
+
+print('List of generated public addresses:', *addresses, sep='\n')
+print()
+
+# Generate internal addresses with custom account index and range.
+addresses = client.generate_addresses(secret_manager, {
+    "accountIndex": 0,
+    "range": {
+        "start": 0,
+        "end": 4,
+    },
+    "internal": True,
+})
+
+print('List of generated internal addresses:', *addresses, sep='\n')
+print()
```

Comment:
Do you really think that this is useful in this example? I think 99% don't care about these and would be fine with just the public addresses and if someone really cares for whatever reason, he should be able to find this option himself

Reply:
I think every example is useful. We should split up examples later on and sort them. But I personally am a big fan if everything is part of examples/how-tos. That's why they exist. And basically no downside

Reply:
> I think every example is useful.

Sure many examples are great, but I think there should be some smaller examples without too much going on for people to get started and then we can have many more examples with more things (they shouldn't be in the same how tos section, otherwise it's also too many examples so you don't even know where to start)
> We should split up examples later on 

Why later and not now already?

Reply:
Because we don't really have the tool yet. So we will need to wait for the next sprint